packages/cdktf-cli/bin/cmds/deploy.ts

```diff
@@ -56,6 +56,8 @@ class Command extends BaseCommand {
         default: false,
       })
       .option("parallelism", {
+        // Note: This parallelism doesn't affect the underlying Terraform traversal parallelism
```

Comment:
We need to add this to the CLI documentation as well, alongside with the explanation in the PR description

Reply:
Quick questions:
- What's the process of updating the CLI documentation
- Which PR description should we update? This one, or the one that added the flag?

Reply:
> What's the process of updating the CLI documentation

This is the file you will need to change within this PR: https://github.com/hashicorp/terraform-cdk/blob/main/website/docs/cdktf/cli-reference/commands.mdx

> Which PR description should we update? This one, or the one that added the flag?

I meant adding the context within this PR description to the explanation you'll add to the documentation